pkg/storaged/stratis-details.jsx

```diff
@@ -641,20 +642,40 @@ export const StratisPoolDetails = ({ client, pool }) => {
         return for_each_async(blockdevs, bd => pool.GrowPhysicalDevice(bd.Uuid));
     }
 
-    let alert = null;
+    const alerts = [];
     if (client.features.stratis_grow_blockdevs &&
         blockdevs.some(bd => bd.NewPhysicalSize[0] && Number(bd.NewPhysicalSize[1]) > Number(bd.TotalPhysicalSize))) {
-        alert = (<Alert key="unused-space"
-                        isInline
-                        variant="warning"
-                        title={_("This pool does not use all the space on its block devices.")}>
+        alerts.push(<Alert key="unused-space"
+                           isInline
+                           variant="warning"
+                           title={_("This pool does not use all the space on its block devices.")}>
             {_("Some block devices of this pool have grown in size after the pool was created. The pool can be safely grown to use the newly available space.")}
             <div className="storage_alert_action_buttons">
                 <StorageButton onClick={grow_blockdevs}>{_("Grow the pool to take all space")}</StorageButton>
             </div>
         </Alert>);
     }
 
+    if (pool.AvailableActions !== "fully_functional") {
+        let alert_msg = null;
+        if (pool.AvailableActions === "no_ipc_requests")
+            alert_msg = _("Some actions are unavailable until the issue is resolved manually.");
```

Comment:
"The issue"? If I was a user I wouldn't know what this means, is there any helpful context here? Also are these errors mutually exclusive?

Reply:
They are mutually exlcusive, I think.

I copied the error message from Stratis here, which also talks about "the issue". I know what "the issue" is in our tests (mismatched LUKS tokens), but I don't know what other issues might cause the "no_ipc_requests" state.

TBH, this is all pretty mysterious to me. I would be happy to just say "Pool is degraded" and not try to decode the specific codes.

Reply:
Pool is degraded sounds good, as there is a link to logs right?

Reply:
Ok! Yes, sounds good to me, too.

Reply:
Done.